### ISSUE TITLE
can't access images

### ISSUE THREAD

**Issue by davep47** [pando85/openeats#5]
Hi, I don't know if you are still working on this or not but I wonder if you could help.
I have installed your branch of openeats on a couple of different servers and run into the same issue each time.
Images upload fine and I can see them in the docker volumes but I can't access the images from the website.
I think the thumbnails are being created but I can't see them or the pictures themselves after they are uploaded.
Any ideas?

**Comment by pando85** [comment]
I just can say that I don't work on it from 3 years ago but I'm still running it and it is working.

This is an extract of my docker-compose.yml:
```yaml
version: '2'

volumes:
  database:
  static_files:
  media_files:

services:
  nginx:
    image: nginx:latest
    restart: unless-stopped
    networks:
      - default
      - proxy
    depends_on:
      - web
    links:
      - web
    volumes_from:
      - web
    volumes:
      - ./nginx/nginx.conf:/etc/nginx/nginx.conf
    expose:
      - "80"

  web:
    image: pando85/openeats:latest
    restart: unless-stopped
    links:
      - database
    expose:
      - "8000"
    environment:
      - PYTHONUNBUFFERED=0 # log into console
      - DEBUG=False
      - TIME_ZONE=Europe/Madrid
      - OETITLE=Recetas Grigri
      - DEFAULT_FROM_EMAIL=Recetas Grigri
    volumes:
      - static_files:/usr/src/app/openeats/static
      - media_files:/usr/src/app/openeats/site-media
    command: ["/usr/local/bin/gunicorn", "openeats.wsgi:application", "-b :8000", "-w 6", "-t 5000"]

  database:
    image: postgres:9.6
    restart: unless-stopped
    volumes:
      - database:/var/lib/postgresql/data
    expose:
      - "5432"

```

This is not the full configuration because I use some network configurations and passwords that I prefer to not post here. 

Also I use this nginx conf:
```conf
user nginx;
worker_processes 4;

error_log  /var/log/nginx/error.log debug;
pid        /var/run/nginx.pid;

events {
    worker_connections  1024;
    # multi_accept on;
}

http {

    ##
    # Basic Settings
    ##

    sendfile on;
    tcp_nopush on;
    tcp_nodelay on;
    keepalive_timeout 65;
    types_hash_max_size 2048;
    # server_tokens off;

    # server_names_hash_bucket_size 64;
    # server_name_in_redirect off;

    include /etc/nginx/mime.types;
    default_type application/octet-stream;

    ##
    # Logging Settings
    ##
    log_format  main  '$remote_addr - $remote_user [$time_local] "$request" '
                      '$status $body_bytes_sent "$http_referer" '
                      '"$http_user_agent" "$http_x_forwarded_for"';

    access_log  /var/log/nginx/access.log  main;

    ##
    # Gzip Settings
    ##

    gzip on;
    gzip_disable "msie6";

    server {

        listen 80 default_server;
        server_name _;
        charset utf-8;
	client_max_body_size 5M;

        location /static {
            alias /usr/src/app/openeats/static;
        }

        location /site-media {
            alias /usr/src/app/openeats/site-media;
        }

        location / {
            proxy_pass http://web:8000;
            proxy_set_header Host $host;
            proxy_set_header X-Real-IP $remote_addr;
            proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
            proxy_connect_timeout       6000;
            proxy_send_timeout          6000;
            proxy_read_timeout          6000;
            send_timeout                6000;
        }

    }
}
```
Hope it helps

**Comment by davep47** [comment]
That did it thanks, The difference between the nginx.conf checked in and yours is 
checked in:
        location /site-media {
            **internal;**
            alias /usr/src/app/openeats/site-media;
        }
yours 
        location /site-media {
            alias /usr/src/app/openeats/site-media;
        }
I did a quick search but didn't find what the "internal" would have been for. 
I figured it had to be something weird because the static images worked fine, even when I copied new files into the static folders it was still able to see them.
Thanks a lot for getting back to me.
